geode-tcp-server/src/test/java/org/apache/geode/distributed/internal/tcpserver/HostAndPortTest.java

```diff
@@ -93,12 +93,6 @@ public void hashCode_of_SocketAddress() {
     assertThat(locator1.hashCode()).isEqualTo(host1address.hashCode());
   }
 
-  @Test
-  public void toString_LocatorAddress() {
-    HostAndPort locator1 = new HostAndPort("fakelocalhost", 8091);
-    assertThat(locator1.toString()).contains("socketInetAddress");
-  }
```

Comment:
I see you modified `toString()`. is there a reason why this test is no longer valuable?

Reply:
There's no contract on HostAndPort.toString() to have this string in its result.  I saw the string rep in test output and didn't like how wordy it was.  It's now HostAndPort[hostname:port].

Reply:
good show